### ISSUE TITLE
Ember 2.0

### ISSUE THREAD

**Issue by lypanov** [minutebase/ember-render-stack#1]
Any chance you guys have a Ember 2.0 variant of this if you're using it actively? Couldn't get my head around content-for like situations in Ember at all until seeing this add-on.

Thank you!

**Comment by lypanov** [comment]
I'll answer myself: https://github.com/minutebase/ember-portal Is actually what I was looking for anyway and is working just fine (some very basic testing at least) for my use case. Very glad I decided to look at your other projects. portal should just be part of Ember!

**Comment by rlivsey** [comment]
I should add a note to the README pointing people at other projects as this one isn't maintained as you've found!

Also, in Ember 2.0 the limitations with outlets which this addon sought to address have been fixed.

I only learned this at EmberCamp last week, and haven't tried it myself yet, but apparently if you have an outlet "foo" with some routes as follows:
- `/` - populate outlet "foo"
- `/bar` - do nothing to outlet
- `/bar/baz` - populate outlet "foo"

In Ember 1.x going from `/` -> `/bar` would work as expected and the outlet would still be populated, but going from `/bar/baz` -> `/bar` would teardown the outlet and not re-populate it with the root's content.

In Ember 2.x it works as expected both ways and going from `/bar/baz` -> `/bar` re-populates the outlet with the root's content.

Thanks.
